tests/e2e/utils/utils.ts

```diff
@@ -102,6 +85,7 @@ export async function waitFor<T>(fn: () => Promise<T>, opts = { timeout: 10000,
 		} catch (err) {
 			lastErr = err;
 			await sleep(interval);
+			elapsed += interval;
```

Comment:
Where is this elapsed coming from?

Reply:
Here's the whole function:
```
export async function waitFor<T>(fn: () => Promise<T>, opts = { timeout: 10000, interval: 100 }) {
	let elapsed = 0;
	let lastErr = undefined;
	const timeout = opts.timeout ?? 1000;
	const interval = opts.interval ?? 100;

	while (elapsed < timeout) {
		try {
			return await fn();
		} catch (err) {
			lastErr = err;
			await sleep(interval);
			elapsed += interval;
		}
	}
	throw lastErr;
}
```

You can see that elapsed is defined on the first line of the function.